geopandas/io/tests/test_sql.py

```diff
@@ -344,6 +344,21 @@ def test_write_postgis_default(self, engine_postgis, df_nybb):
         df = read_postgis(sql, engine, geom_col="geometry")
         validate_boro_df(df)
 
+    def test_write_postgis_uppercase_tablename(self, engine_postgis, df_nybb):
+        """Tests writing GeoDataFrame to PostGIS with uppercase tablename."""
+        engine = engine_postgis
+        table = "Nybb"
```

Comment:
I think that this conflicts with `nybb` used below. Can you use a different (unique) name for this test?

Reply:
I guess only the schema tests failed cause COPY did not have schema as a separate keyword in quotes, Everything should pass now 🤞